dockerfiles/theia-endpoint-runtime/Dockerfile

```diff
@@ -10,34 +10,21 @@
 
 FROM ${BUILD_ORGANIZATION}/${BUILD_PREFIX}-theia-dev:${BUILD_TAG} as builder
 
+# Export GITHUB_TOKEN into environment variable
+ARG GITHUB_TOKEN=''
+ENV GITHUB_TOKEN=$GITHUB_TOKEN
 
-# define in env variable GITHUB_TOKEN only if it is defined
-# else check if github rate limit is enough, else will abort requiring to set GITHUB_TOKEN value
-ARG GITHUB_TOKEN
-
-# Check github limit
-RUN if [ ! -z "${GITHUB_TOKEN-}" ]; then \
-        export GITHUB_TOKEN=$GITHUB_TOKEN; \
-        echo "Setting GITHUB_TOKEN value as provided"; \
-    else \
-        export GITHUB_LIMIT=$(curl -s 'https://api.github.com/rate_limit' | jq '.rate .remaining'); \
-        echo "Current API rate limit https://api.github.com is ${GITHUB_LIMIT}"; \
-        if [ "${GITHUB_LIMIT}" -lt 10 ]; then \
-            printf "\033[0;31m\n\n\nRate limit on https://api.github.com is reached so in order to build this image, "; \
-            printf "the build argument GITHUB_TOKEN needs to be provided so build will not fail.\n\n\n\033[0m"; \
-            exit 1; \
-        else \
-            echo "GITHUB_TOKEN variable is not set but https://api.github.com rate limit has enough slots"; \
-        fi \
-    fi
-
-#invalidate cashe
-ADD https://${GITHUB_TOKEN}:x-oauth-basic@api.github.com/repos/theia-ide/theia/git/refs/head /tmp/branch_info.json
-ADD https://${GITHUB_TOKEN}:x-oauth-basic@api.github.com/repos/eclipse/che-theia/git/refs/head /tmp/branch_info.json
+# Invalidate cashe
+ADD https://api.github.com/rate_limit /tmp/github_rate_limit.json
```

Comment:
we should invalidate cache only if remote code has changed, not if rate limit is changing

Reply:
Yes, you are right, will revert cache invalidation lines

Reply:
@benoitf done. Thanks for the remark.